mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -53,14 +54,16 @@
  */
 public class MongoDbClient extends DB {
 
-    /** Update options to do an upsert. */
-    private static final UpdateOptions UPSERT = new UpdateOptions()
-            .upsert(true);
-
     /** Used to include a field in a response. */
     protected static final Integer INCLUDE = Integer.valueOf(1);
 
-    /** The database name to access. */
+    /** The options to use for inserting many documents */
+    protected static final InsertManyOptions INSERT_MANY_OPTIONS =
+    new InsertManyOptions().ordered(false);
```

Comment:
Can this be renamed to show what it stands for instead of the type of object it is. Maybe INSERT_UNORDERED?